Readability.js

```diff
@@ -410,6 +410,14 @@ Readability.prototype = {
     // Check if any "dir" is set on the toplevel document element
     this._articleDir = doc.documentElement.getAttribute("dir");
 
+    //helper function used below in the 'while' loop:
+    function purgeNode(node, allElements) {
+      for (let i = node.childNodes.length; --i >= 0;) {
```

Comment:
Please use 'var' instead of 'let' (I know you used 'let' in your original patch, but since then we've decided to make this shared script work in a more generic web context, where 'let' isn't supported).